docs/src/pages/guides/understand-mui-packages/understand-mui-packages.md

```diff
@@ -34,35 +34,48 @@ For this reason, abstracting into smaller packages not only allows MUI to grow o
 
 The packages can be categorized into 3 layers, as shown in the picture below:
 
-<img src="/static/images/packages/mui-packages.png" style="width: 700px; max-width: 100%;" />
+<img src="/static/images/packages/mui-packages.png" style="display: block; width: 448px; max-width: 100%; margin: auto;" />
 
 Let's take a look at each layer to understand how they work together, starting from the bottom:
 
+> 💡 Glossary
+>
+> **install** refers to running `yarn add $module` or `npm install $module`.
+>
+> **import** refers to making a module API available in your code by adding `import ... from '$module'`.
```

Comment:
Would have a ul > li here help? I personally got confused the first time I read this section:

<img width="451" alt="Screenshot 2021-11-10 at 12 01 29" src="https://user-images.githubusercontent.com/3165635/141101526-24b6d19e-b050-4d29-aaf1-80262ded0f2b.png">

Reply:
👍